File: Level-1/32.java
class Solution {
    public int solution(int[] nums) {
        int answer = 0;
        
        for(int i = 0; i < nums.length; i++){
            for(int j = i + 1; j < nums.length; j++){
                for(int k = j + 1; k < nums.length; k++){
                    int num = nums[i] + nums[j] + nums[k];
                    int cnt = 0;
                    
                    for(int l = 2; l < num; l++){
                        if(num % l == 0) cnt++;
                    }
                    
                    if(cnt == 0) answer++;
                }
            }
        }
        
        return answer;
    }
}

File: Level-1/32.js
function check(n) {
    for(let i = 2; i <= Math.sqrt(n); i++) {
        if(n % i === 0) return false;
    }
    
    return true;
}

function solution(nums) {
    let cnt = 0
    
    for(let i = 0; i < nums.length - 2; i++) {
        for(let j = i + 1; j < nums.length - 1; j++) {
            for(let k = j + 1; k < nums.length; k++) {
                check(nums[i] + nums[j] + nums[k]) ? cnt++ : '';
            }
        }
    }
    
    return cnt;
}
